### PR TITLE
support alert forward from *KS.

### PR DESCRIPTION
**Pull request by morvencao**
ref: open-cluster-management/backlog#15086
Signed-off-by: morvencao <lcao@redhat.com>

### CODE REVIEW

**Comment by morvencao** [comment]
We can receive the alert from GKE now:

```
# oc exec -it observability-alertmanager-0 -c alertmanager -- bash
bash-4.4$ amtool --alertmanager.url http://localhost:9093 alert -o json
[{"annotations":{"description":"Cluster has overcommitted CPU resource requests for Pods and cannot tolerate node failure.","summary":"Cluster has overcommitted CPU resource requests."},"endsAt":"2021-08-24T10:15:02.969Z","fingerprint":"26741a84f4ddfde7","receivers":[{"name":"null"}],"startsAt":"2021-08-24T09:54:32.969Z","status":{"inhibitedBy":[],"silencedBy":[],"state":"active"},"updatedAt":"2021-08-24T10:11:02.986Z","generatorURL":"http://prometheus-k8s-0:9090/graph?g0.expr=sum%28namespace_cpu%3Akube_pod_container_resource_requests%3Asum%29+%2F+sum%28kube_node_status_allocatable%7Bresource%3D%22cpu%22%7D%29+%3E+%28count%28kube_node_status_allocatable%7Bresource%3D%22cpu%22%7D%29+-+1%29+%2F+count%28kube_node_status_allocatable%7Bresource%3D%22cpu%22%7D%29\u0026g0.tab=1","labels":{"alertname":"KubeCPUOvercommit","cluster_type":"GKE","severity":"warning"}}]
```

Currently external label `cluster_type: GKE` to identify the alert from that GKE cluster, because `clusterID` is not available in GKE clusters.

File: operators/endpointmetrics/controllers/observabilityendpoint/observabilityaddon_controller.go [hunk 1]
```diff
@@ -226,6 +225,13 @@ func (r *ObservabilityAddonReconciler) initFinalization(
 		if err != nil {
 			return false, err
 		}
+
+		// revert the change to cluster monitoring stack
+		err = revertClusterMonitoringConfig(ctx, r.Client, installPrometheus)
```
Comment (clyang82) [inline operators/endpointmetrics/controllers/observabilityendpoint/observabilityaddon_controller.go:230]: why move it out of `if !installPrometheus {`? Do we need to `revertClusterMonitoringConfig` for *KS?

Comment (morvencao) [reply operators/endpointmetrics/controllers/observabilityendpoint/observabilityaddon_controller.go:230]: we don't need to create `cluster-monitoring-config` configmap for *KS, but we need to create the two secrets: `hub-alertmanager-router-ca` and `observability-alertmanager-accessor`. That's why I add this.

**Pull request merged**